encrypted-media-respec.html

```diff
@@ -344,14 +342,22 @@
               <li><p>XORing the origin with a fixed value (because it is trivially reversible)</p></li>
             </ul>
           </div>
+          <p>
+            <a def-id="distinctive-identifiers"></a> may be considered <a def-id="non-associable"></a> as long as such correlation or association is only possible by the entity, such as an <a href="#privacy-individualization">individualization</a> server, that originally generated the <a def-id="distinctive-identifiers"></a> value.
```

Comment:
This is a little confusing because the definition of "associable" is absolute: something is either associable or not. Distinctive Identifiers certainly meet the definition of "associable" but here we say they may be considered "non-associable".

Perhaps we need to refine the definition of "associable" to be "associable by an entity" (i.e. "two identifiers are associable by an entity if that entity can ..."). Then we will have:
- An identifier is 'associable' if it is associable by any entity.
- Distinctive Identifiers are associable identifiers that are not associable by the application